src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,96 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * <h1>TaskCommand Class</h1>
+ * This command records the description of the task and indicate whether the programme is done or not.
+ *
+ * @author Eng Xuan En
+ */
+public class Task {
+    protected String description;
```

Comment:
Is there a reason why you would want these to be 'protected' instead of 'private'?